src/machines/java/com/enderio/machines/common/blockentity/sync/LargeMachineEnergyDataSlot.java

```diff
@@ -23,7 +24,7 @@ public LargeMachineEnergyDataSlot(Supplier<IMachineEnergyStorage> getter, Consum
 
     @Override
     public Tag serializeValueNBT(IMachineEnergyStorage value) {
-        ILargeMachineEnergyStorage storage = (ILargeMachineEnergyStorage) getter.get();
+        ILargeMachineEnergyStorage storage = (ILargeMachineEnergyStorage) getter.get(); //why the getter?
```

Comment:
Likely an artefact of when I ported this to the new dataslot stuff, you should be able to replace the `getter.get()` with `value`